qml/FileListPage.qml

```diff
@@ -146,6 +159,20 @@ Page {
                                            : qsTrId("sailfish-office-me-hide_search")
                 onClicked: page.searchEnabled = !page.searchEnabled
             }
+            
+            MenuItem {
+                //% "Sort by: %1"
+                text: qsTrId("sailfish-office-me-sort_by").arg(getSortParameterName(filteredModel.sortParameter))
```

Comment:
By default you now get "Sort by:" without any actual sorting noted. Should be always sorted by something.

Reply:
Ok. As @dcaliste told, I'll sort it by date by default

Reply:
Done

Reply:
Done